.github/workflows/revert-web-wiki-updates.yml

```diff
@@ -0,0 +1,26 @@
+name: Revert Wiki Changes Made Through the Web Interface
+on:
+  gollum
+
+jobs:
+  revert:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{github.repository_owner}}/oppia.wiki
+          token: ${{secrets.WIKI_ACCESS_TOKEN}}
+          fetch-depth: 0
+      - name: Add remote
+        run: git remote add source https://github.com/${{github.repository_owner}}/oppia-web-developer-docs.git
+      - name: Fetch remote
+        run: git fetch source
+      - name: Revert commits
+        run: |
+          git config user.name ${{github.actor}}
+          git config user.email ${{github.actor}}@users.noreply.github.com
+          git revert --no-edit source/develop..origin/master
+      - name: Push to deployment repository
+        run: git push source master:develop
```

Comment:
is this pushing to a source, not master? (not sure!) is there any way to test this workflow before merging?

Reply:
This pushes the local `master` branch to the `develop` branch on the remote `source`. For context, the wiki is served from the `master` branch, and I haven't been able to find a way to change which branch is used by GitHub to render the wiki. On the source repo, we want to have the main branch be called `develop` to align with how we name branches in `oppia/oppia`

Regarding testing, see the proof of correctness:

> This workflow has been tested on https://github.com/U8NWXD/test-oppia-docs (which mimics `oppia/oppia-web-developer-docs`) and https://github.com/U8NWXD/test-oppia (which mimics `oppia/oppia`).